services/web/pkg/config/options.go

```diff
@@ -26,7 +26,10 @@ type Options struct {
 	AccessDeniedHelpURL      string           `json:"accessDeniedHelpUrl,omitempty" yaml:"accessDeniedHelpUrl" env:"WEB_OPTION_ACCESS_DENIED_HELP_URL" desc:"Specifies the target URL valid for the ocis instance for the generic logged out / access denied page."`
 	TokenStorageLocal        bool             `json:"tokenStorageLocal" yaml:"tokenStorageLocal" env:"WEB_OPTION_TOKEN_STORAGE_LOCAL" desc:"Specifies whether the access token will be stored in the local storage when set to 'true' or in the session storage when set to 'false'. If stored in the local storage, login state will be persisted across multiple browser tabs, means no additional logins are required. Defaults to 'true'."`
 	DisabledExtensions       []string         `json:"disabledExtensions,omitempty" yaml:"disabledExtensions" env:"WEB_OPTION_DISABLED_EXTENSIONS" desc:"Disable specific Web extensions identified by their ID. The ID can e.g. be taken from the 'index.ts' file of the web extension. Multiple ID's can be written as comma separated list like: 'com.github.owncloud.web.files.search','com.github.owncloud.web.files.print'. See the Environment Variable Types documentation for details."`
+	Mode                     string           `json:"mode,omitempty" yaml:"mode" env:"WEB_OPTION_MODE" desc:"Defines the mode in which Web will be running. Currently, the only supported option is 'embed', which will enable a stripped down version of Web with reduced functionality. This can be used to integrate Web into other applications, e.g. via iFrame. When no specific mode is given, Web will run as usual with all functionality enabled. This is also the default behaviour."`
+	EmbedTarget              string           `json:"embedTarget,omitempty" yaml:"embedTarget" env:"WEB_OPTION_EMBED_TARGET" desc:"Defines how Web is being integrated when running in embed mode. Currently, the only supported option is 'location', which will run embedded Web as location picker. Resource selection will be disabled and the selected resources array always includes the current folder as the only item. When no specific target is given, Web will run regular as defined via the 'mode' config. This is also the default behaviour.'"`
```

Comment:
```suggestion
	EmbedTarget              string           `json:"embedTarget,omitempty" yaml:"embedTarget" env:"WEB_OPTION_EMBED_TARGET" desc:"Defines how Web is being integrated when running in 'embed' mode. Currently, the only supported options are '' (empty) and 'location'.  With '' which is the default, Web will run regular as defined via the 'mode' config. With 'location', Web will run embedded Web as location picker. Resource selection will be disabled and the selected resources array always includes the current folder as the only item."`
```
I am not clear about `...always includes the current folder as the only item...`. What does that mean?